Makefile.in

```diff
@@ -257,3 +271,9 @@ ifneq ($(strip $(findstring TAGS.emacs,$(MAKECMDGOALS)) \
   CFG_INFO := $(info cfg: including ctags rules)
   include $(CFG_SRC_DIR)mk/ctags.mk
 endif
+
+.DEFAULT:
```

Comment:
What does this end up doing? Is this just to translate `make foo` into a help message here?